packages/editor/src/functions/sceneFunctions.tsx

```diff
@@ -103,9 +104,7 @@ export const saveSceneGLTF = async (
 
     return
   }
-  const result = await Engine.instance.api
-    .service(assetPath)
-    .create({ name: sceneNameWithoutExtension, assetURL, project: projectName })
+  const result = await Engine.instance.api.service(assetPath).create({ assetURL, project: projectName })
```

Comment:
This should have `isScene: true` in data argument. When saving a scene using `save as`, a new scene is created but before it can update manifest.json, it exits in `createSceneFiles` hook in `asset.hooks` on line 230. As `isScene` is null, resulting in the creation of a new scene without the manifest.json updating